app/controllers/participants_controller.rb

```diff
@@ -80,42 +81,42 @@ def handle_import
       success.empty? ? () : (flash[:success] = "Imported participants: <br/>" + success)
     end
   end
-  
+
   def handle_simple
```

Comment:
This is good case handling, I like the idea of defaulting to 1 match degree! (at least that's what I think this does?)

Reply:
Yep!